### DISCUSSION TITLE
+/-level-colors seemingly doesn't work?

### DISCUSSION THREAD

**Discussion by Walaryne** [ImageMagick/ImageMagick#4135] [Help]
Upon trying to use + or -level-colors, I just get this error.
What am I doing wrong?

```
magick convert 1_RIM2.png -level_colors green,black result.png
convert: unrecognized option `-level_colors' @ error/convert.c/ConvertImageCommand/2141.
```
Here's my `--version` output:
```
Version: ImageMagick 7.1.0-2 Q16 x86_64 2021-06-25 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(4.5) 
Delegates (built-in): bzlib cairo djvu fontconfig freetype heic jbig jng jp2 jpeg lcms lqr ltdl lzma openexr pangocairo png raqm raw rsvg tiff webp wmf x xml zip zlib
```

**Comment by fmw42** [comment]
+level is for converting gray to color. -level is for adjusting intensities. In IM 7, use magick, not magick convert in place of convert. Use hyphen not underscore. Assuming you have a color image, you need to convert to grayscale (-colorspace gray) before using +level-colors in most use cases. Normally one would just start with a grayscale image. However, you can use a color image and not convert to grayscale.

`magick 1_RIM2.png +level-colors green,black result.png`

**Reply by Walaryne** [reply]
OH thanks, I didn't realize I was typing it wrong, my bad. My goal was to try and change the colors of images that have edge blending/aliasing. It's the only function I know of that can really do this seamlessly.
